SerialPrograms/Source/CommonFramework/VideoPipeline/Backends/CameraWidgetQt6.5.cpp

```diff
@@ -275,9 +275,11 @@ void CameraSession::connect_video_sink(QVideoSink* sink){
                 WriteSpinLock lg(m_frame_lock);
 
                 //  Skip duplicate frames.
+                #ifndef __APPLE__

```

Comment:
Better idea: Skip the check is the timestamp is invalid instead of if-def'ing on the platform.